gateway/1.4/modules/ROOT/pages/flex-troubleshoot-requests.adoc

```diff
@@ -6,12 +6,16 @@ endif::[]
 
 When configuring and running Flex Gateway, you might encounter connection issues. 
 
-You can troubleshoot the request connection between the API upstream and Flex Gateway, as well as the connection between Flex Gateway and the client, by isolating the connections. Isolating Flex Gateway connections allows you to localize the issue.
+You can troubleshoot the outbound connection between the upstream service and Flex Gateway, as well as the inbound connection between Flex Gateway and the client, by making direct requests. Making direct requests isolates Flex Gateway connections allowing you to localize the issue. 
 
-== Isolate Connection Between the API Upstream and Flex Gateway
-Isolating the connection between the API upstream and Flex Gateway allows you to test for connection issues between the two.
+The following diagram demonstrates the difference between inbound and outbound requests. 
 
-To isolate the connection between the API upstream and Flex Gateway:
+image:flex-inbound-outbound-requests.png[This image describes how inbound connections are between the client and Flex Gateway and outbound connections are between Flex Gateway and the upstream service, align=center]
```

Comment:
No need to mention "image" in the alt text.

Inbound connections between the client and Flex Gateway and outbound connections between Flex Gateway and the upstream service.